test/test_onboard_service.sh

```diff
@@ -4,9 +4,24 @@
 source test/utils.sh
 
 KEPTN_EXAMPLES_BRANCH=${KEPTN_EXAMPLES_BRANCH:-"master"}
+KEPTN_NAMESPACE=${KEPTN_NAMESPACE:-keptn}
+KEPTN_API_TOKEN=$(kubectl get secret keptn-api-token -n "$KEPTN_NAMESPACE" -o jsonpath='{.data.keptn-api-token}' | base64 --decode)
 
 echo "Testing onboarding for project $PROJECT"
 
+echo "Using remote execution plane services: ${REMOTE_EXECUTION_PLANE}"
+
+# check if REMOTE_EXECUTION_PLANE is set to true. If yes, scale down the helm-service and jmeter in the keptn namespace and install the services via their helm charts
+if [[ "${REMOTE_EXECUTION_PLANE}" == "true" ]]; then
+  kubectl scale deployment/helm-service -n "${KEPTN_NAMESPACE}" --replicas=0
+  kubectl scale deployment/jmeter-service -n "${KEPTN_NAMESPACE}" --replicas=0
+
+  KEPTN_API_HOSTNAME=$(kubectl -n "${KEPTN_NAMESPACE}" get service api-gateway-nginx -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+  KEPTN_VERSION_TAG=$(kubectl get deployment -n "${KEPTN_NAMESPACE}" helm-service -o go-template='{{index .metadata.labels "app.kubernetes.io/version"}}')
+
+  helm install helm-service ./helm-service/chart -f ./helm-service/chart/values.yaml -n keptn-helm-service --set helmservice.image.tag="${KEPTN_VERSION_TAG}" --set distributor.image.tag="${KEPTN_VERSION_TAG}" --set remoteControlPlane.enabled=true --set remoteControlPlane.api.protocol=http --set remoteControlPlane.api.hostname="${KEPTN_API_HOSTNAME}" --set remoteControlPlane.api.token="${KEPTN_API_TOKEN}" --create-namespace
+  helm install jmeter-service ./jmeter-service/chart -f ./jmeter-service/chart/values.yaml -n keptn-jmeter-service --set jmeterservice.image.tag="${KEPTN_VERSION_TAG}" --set distributor.image.tag="${KEPTN_VERSION_TAG}" --set remoteControlPlane.enabled=true --set remoteControlPlane.api.protocol=http --set remoteControlPlane.api.hostname="${KEPTN_API_HOSTNAME}" --set remoteControlPlane.api.token="${KEPTN_API_TOKEN}" --create-namespace
```

Comment:
Please use the charts provided within the CI build artifact `keptn-installer`. They *should* be called
* `dist/keptn-installer/helm-service-${KEPTN_VERSION_TAG}.tgz`
* `dist/keptn-installer/jmeter-service-${KEPTN_VERSION_TAG}.tgz`

see https://github.com/keptn/keptn/suites/2202706220/artifacts/45499321 as a reference

In addition, you should not have to specify those tags (they are specified directly in the chart that's available via the CI build artifact) please remove
* `--set helmservice.image.tag="${KEPTN_VERSION_TAG}" --set distributor.image.tag="${KEPTN_VERSION_TAG}"`
* `--set jmeterservice.image.tag="${KEPTN_VERSION_TAG}" --set distributor.image.tag="${KEPTN_VERSION_TAG}"`